tests/acceptance/features/coreApiWebdavMove1/moveFolder.feature

```diff
@@ -287,14 +287,28 @@ Feature: move (rename) folder
     When user "Alice" moves folder "testFolder" to "<folder-name>" using the WebDAV API
     Then the HTTP status code should be "<http-status-code>"
     Examples:
-      | dav-path-version | folder-name   | http-status-code |
-      | old              | /.            | 409              |
-      | old              | /..           | 404              |
-      | new              | /.            | 409              |
-      | new              | /..           | 404              |
+      | dav-path-version | folder-name | http-status-code |
+      | old              | /.          | 409              |
+      | old              | /..         | 404              |
+      | new              | /.          | 409              |
+      | new              | /..         | 404              |
 
     @skipOnRevaMaster
     Examples:
-      | dav-path-version | folder-name   | http-status-code |
-      | spaces           | /.            | 409              |
-      | spaces           | /..           | 400              |
\ No newline at end of file
+      | dav-path-version | folder-name | http-status-code |
+      | spaces           | /.          | 409              |
+      | spaces           | /..         | 400              |
+
+
+  Scenario Outline: rename a folder to .htaccess
+    Given using <dav-path-version> DAV path
+    And user "Alice" has created folder "/testshare"
+    When user "Alice" moves folder "/testshare" to "/.htaccess" using the WebDAV API
+    Then the HTTP status code should be "201"
```

Comment:
https://github.com/owncloud/ocis/issues/1345#issuecomment-577534595
>this is a feature. ocis properly handles .htaccess because there is no php running inside apache that could get be influenced. File uploads are handled with dedicated data services that will make the file available to the storage provider when the upload is done. Atomicity depends on the implementation.